spec/system/provider_interface/provider_responds_to_application_spec.rb

```diff
@@ -15,22 +16,64 @@
     create(:submitted_application_choice, status: 'rejected', rejected_at: Time.zone.now, course_option: course_option)
   end
 
-  scenario 'Provider cannot respond to an application without make_decisions' do
+  scenario 'Provider cannot respond to an application if user lacks make_decisions' do
     given_i_am_a_provider_user_with_dfe_sign_in
     and_i_am_permitted_to_see_applications_for_my_provider
     and_i_am_not_permitted_to_make_decisions_for_my_provider
+    and_my_organisation_is_permitted_to_make_decisions
     and_i_sign_in_to_the_provider_interface
 
     when_i_try_to_respond_to_an_application
     then_i_get_access_denied
   end
 
-  scenario 'Provider can respond to an application currently awaiting_provider_decision' do
+  scenario 'Provider cannot respond to an application if organisation lacks make_decisions' do
```

Comment:
This system spec now contains several scenarios, which isn't how we do system specs. Could we break this up and/or find a way to test the edges at a lower level?

Reply:
The way `#can_make_offer?` behaves has been thoroughly tested in the relevant service spec. What's missing is a way of asserting controller flows in the absence or presence of these feature flags. I'll remove these scenaria from here and create a new 'provider responds to application' file for post-launch (all feature flags on).